junit/src/main/java/org/kaazing/k3po/junit/rules/SpecificationStatement.java

```diff
@@ -112,10 +122,11 @@ public void evaluate() throws Throwable {
                 }
             }
 
+            // Stefan - not sure what the comment bellow is about. There is no join() method in the statement or rule class.
```

Comment:
With some help from Costin, I have found an older commit (actually two) where a method called join was removed and replaced with start and finish. Take a look at them:
https://github.com/k3po/k3po/commit/eeb9d4127edcea5216f1c5cd94263f873c33b910
https://github.com/k3po/k3po/commit/e44f2e8b0b782faafb73edfdcd04d95d41f630e0